demo/server/src/modules/demo-data/Workorders.ts

```diff
@@ -173,24 +174,39 @@ const workorders = [
   }
 ];
 
+function updateWorkorderDates() {
+  const today = new Date();
+  const tomorrow = new Date();
+  tomorrow.setDate(today.getDate() + 4);
```

Comment:
That is no longer needed as  I have disabled this fields. They are provided by step when starting